doc/install_urcap.md

```diff
@@ -0,0 +1,31 @@
+# How to install the RS485 URCap
```

Comment:
@urrsk Is there something like an "official" guide to install a URCap? It might make more sense to link to one instead of creating on in each repository that we create (see [ros_as_a_service](https://github.com/UniversalRobots/Universal_Robots_ROS_as_a_Service_URCap/blob/master/doc/installation.md), [ur_robot_driver (external_control)](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/blob/master/ur_robot_driver/doc/install_urcap_e_series.md))

Reply:
It is described in the robot manual!

Reply:
but this is unfortunately not really referencable from an online documentation, right? Sure, we can point to UR's download page, which requires a login, let the user search for the manual and then the respective chapter.

But linking to a specific online documentation directly, would be much more beneficial in my opinion.

Reply:
Yes, fully agree with @fmauch.

UR support being put behind a login was a rather unwelcome surprise ..

Reply:
You can now read the manual online on http://help.universal-robots.com/ without the need to sign in